src/status_im2/common/home/view.cljs

```diff
@@ -30,84 +29,65 @@
-        new-notifications?     (pos? notif-count)
-        notification-indicator :unread-dot
-        counter-label          "0"]
-    [rn/view {:style (assoc style :height 56)}
-     ;; Left Section
-     [rn/touchable-without-feedback {:on-press #(rf/dispatch [:navigate-to :my-profile])}
-      [rn/view
-       {:accessibility-label :open-profile
-        :style               {:position :absolute
-                              :left     20
-                              :top      12}}
-       [quo/user-avatar
-        (merge
-         {:status-indicator? true
-          :size              :small}
-         avatar)]]]
-     ;; Right Section
-     [rn/view
-      {:style {:position       :absolute
-               :right          20
-               :top            12
-               :flex-direction :row}}
-      (when-not hide-search
-        [base-button :i/search #() :open-search-button button-common-props])
-      [base-button :i/scan #() :open-scanner-button button-common-props]
-      [base-button :i/qr-code #() :show-qr-button button-common-props]
-      [rn/view                     ;; Keep view instead of "[:<>" to make sure relative
-       ;; position is calculated from this view instead of its parent
-       [hole-view/hole-view
-        {:key   new-notifications? ;; Key is required to force removal of holes
-         :holes (cond
-                  (not new-notifications?) ;; No new notifications, remove holes
-                  []
-
-                  (= notification-indicator :unread-dot)
-                  [{:x 37 :y -3 :width 10 :height 10 :borderRadius 5}]
-
-                  :else
-                  [{:x 33 :y -7 :width 18 :height 18 :borderRadius 7}])}
-        [base-button :i/activity-center #(rf/dispatch [:activity-center/open])
-         :open-activity-center-button button-common-props]]
-       (when new-notifications?
-         (if (= notification-indicator :counter)
-           [quo/counter
-            {:accessibility-label :notifications-unread-badge
-             :outline             false
-             :override-text-color colors/white
-             :override-bg-color   colors/primary-50
-             :style               {:position :absolute
-                                   :left     34
-                                   :top      -6}}
-            counter-label]
-           [rn/view
-            {:accessible          true
-             :accessibility-label :notifications-unread-badge
-             :style               {:width            8
-                                   :height           8
-                                   :border-radius    4
-                                   :top              -2
-                                   :left             38
-                                   :position         :absolute
-                                   :background-color colors/primary-50}}]))]]]))
+  [rn/view {:style (style/top-nav-container style)}
```

Comment:
I think `{:style (style/top-nav-container style)}` looks weird (passing a style prop to a style), maybe `(merge style/top-nav-container style)` would be more self-explainatory